external/avro/src/main/scala/org/apache/spark/sql/avro/package.scala

```diff
@@ -52,12 +50,12 @@ package object avro {
    * @since 3.0.0
    */
   @Experimental
+  @deprecated("Please use 'org.apache.spark.sql.avro.functions.from_avro' instead.", "3.0.0")
```

Comment:
This is since 3.0.0. So I think we don't need to deprecate it but just remove it?

Reply:
It's added in 2.4 and deprecated in 3.0. Why remove it?

Reply:
There is `@since 3.0.0` tag. I think it means this API is added in 3.0.0.

Reply:
That's a question from my side because I would read `deprecated since 3.0`. If you're 100% sure then I've learned something.

Reply:
It's correct to remove it out. Thanks for pointing this out.

Reply:
I've just checked scala source:
```
 *  @param  since   a string identifying the first version in which the definition was deprecated
```

Reply:
3.0.0 is not released. Therefore, technically we don't have the first version.